pages/pipelines/input_step.md.erb

```diff
@@ -0,0 +1,223 @@
+# Input Step
+
+An *input* step is used to collect information from a user. 
+
+<%= toc %>
+
+Input steps with can only be defined using a [`pipeline.yml`](/docs/pipelines/defining-steps) file.
+
+You can add form fields to input steps with the `fields` attribute. There are two field types available: text or select. The select input type is displayed as radio buttons when it contains less than six items, and as a drop down list when it contains more than six items.    
+
+The data you collect from these fields is then available to subsequent steps via the [build meta-data](/docs/pipelines/build-meta-data).
+
+## Input Step Attributes
+
+_Optional attributes:_
+
+<table>
+  <tr>
+    <td><code>prompt</code></td>
+    <td>
+      The instructional message displayed in the dialog box when the step is activated.<br>
+      <em>Example:</em> <code>"Release to production?"</code><br>
+      <em>Example:</em> <code>"Fill out the details for this release"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>fields</code></td>
+    <td>
+      A list of input fields required to be filled out before the step will be marked as passed.<br>
+      Available input field types: <code>text</code>, <code>select</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>branches</code></td>
+    <td>
+      The <a href="/docs/pipelines/branch-configuration#branch-pattern-examples">branch pattern</a> defining which branches will include this input step in their builds.<br>
+      <em>Example:</em> <code>"master stable/*"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>if</code></td>
+    <td>
+      A boolean expression to restrict the running of the step. See <a href="/docs/pipelines/conditionals">Using conditionals</a> for supported expressions.<br>
```

Comment:
Maybe a note for the existing block step page too, but "the running of the step" feels a bit off here? 🤷🏻‍♂️

Reply:
Yeah, it's totally awkward. I've come up with some alternates, would love to get your feels on them!

A boolean expression:
to restrict when a step runs
to restrict when this step is added to a build
to restrict when the step is run in the pipeline
to define the circumstances when this step will be added to a build from this pipeline
to limit the step to only run under specific circumstances
to prevent the step from being added to the build under certain conditions

Reply:
How about something like "that [omits/disables/ignores] the step when false"?

Reply:
Yeah cool. I'll do the update for that in the conditionals update PR I've got on the go 👌🏻